packages/api/src/db/comment.ts

```diff
@@ -0,0 +1,68 @@
+import {RichTextNode} from '../graphql/richText'
+
+export enum CommentStatus {
+  Approved = 'approved',
+  PendingApproval = 'pendingApproval',
+  PendingUserChanges = 'pendingUserChanges',
+  Rejected = 'rejected'
+}
+
+export enum CommentRejectionReason {
+  Misconduct = 'misconduct',
+  Spam = 'spam'
+}
+
+export enum CommentAuthorType {
+  Admin = 'admin',
+  ArticleAuthor = 'articleAuthor',
+  VerifiedUser = 'verifiedUser',
+  PeerUser = 'peerUser',
+  Journalist = 'journalist',
+  Moderator = 'moderator'
+}
+
+export interface CommentData {
+  siteID?: string
+  userID: string
+  peerID?: string
+  permalink?: string
+
+  // comment could be on different types of data models
+  articleID?: string
+  imageID?: string
```

Comment:
You can replace `imageID` with `pageID`. But is that smart? If you make them all optional it means we can create comments that belong to neither an article nor a page. Maybe something like an `ownerID` and `ownerType` would make more sense. Maybe we can find a better word then owner.

Reply:
I have changed it now to be `itemId` and without a type enum. what do you think?